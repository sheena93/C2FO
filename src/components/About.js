/**
 * @component About
 * This component is a stateless component to show the about page
 * @return {element} returns About component
 */

import React from 'react';

const About = () => {
  return (
    <div className='container'>
      <h5>About Us</h5>
      <p>
        This Application show the book list with add and delete functionality.
      </p>
    </div>
  );
};

export default About;