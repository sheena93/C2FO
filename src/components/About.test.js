import React from "react";
import {render,cleanup} from '@testing-library/react'
import About from './About';

afterEach(cleanup)

describe('About', () => {
    it('should take a snapshot', () => {
        const { asFragment } = render(<About />)
        expect(asFragment(<About />)).toMatchSnapshot()
       })
});



