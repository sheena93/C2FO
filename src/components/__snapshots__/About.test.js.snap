// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h5>
      About Us
    </h5>
    <p>
      This Application show the book list with add and delete functionality.
    </p>
  </div>
</DocumentFragment>
`;
